doc/newsboat.asciidoc

```diff
@@ -1168,6 +1168,7 @@ Identifier:Meaning
 |======================================================================
 Identifier:Meaning
 [[feedlist-title-format-T]]<<feedlist-title-format-T,+T+>>:Currently selected tag (empty if none selected)
+[[feedlist-title-format-U]]<<feedlist-title-format-U,+U+>>:Number of unread articles
```

Comment:
The description for `%u` above also says "number of unread articles", so now the tables are confusing. Can you please move `%u` (and also `%t` and maybe `%F`) out of the above table and into this and the next one, giving them accurate descriptions so there is no confusion over what they actually mean?

Reply:
Good idea, done: https://github.com/newsboat/newsboat/pull/1496/commits/9f93fcd5f33a9fb4e3c881fcba6cd660ced6817a